models/issue.go

```diff
@@ -430,6 +430,10 @@ func (issue *Issue) PatchURL() string {
 	return ""
 }
 
+func (issue *Issue) ReferenceLink() string {
+	return fmt.Sprintf("%s#%d", issue.Repo.FullName(), issue.Index)
+}
+
```

Comment:
If this reference link is used only once, I would suggest it to be constructed in the template directly, to keep the `issue.go` simple and clear.

Reply:
It's still a niche method, but it's used twice - once in the copy-paste text handler, the other in the actual message - though idk if it should still be removed as a function and replaced with strings in both locations